src/main/java/org/jabref/gui/customjfx/CustomJFXPanel.java

```diff
@@ -61,18 +61,19 @@ private void sendInputMethodEventToFX(InputMethodEvent e) {
             insertionIndex = e.getCaret().getInsertionIndex();
         }
 
-        EmbeddedSceneInterface myScencePeer = null;
-        try {
-            //the variable must be named different to the original, otherwise reflection does not find the right ones
-            myScencePeer = (EmbeddedSceneInterface) scenePeerField.get(this);
-        } catch (IllegalArgumentException | IllegalAccessException ex) {
-            LOGGER.error("Could not access scenePeer Field", ex);
-        }
-
-        myScencePeer.inputMethodEvent(
-                javafx.scene.input.InputMethodEvent.INPUT_METHOD_TEXT_CHANGED,
-                InputMethodSupport.inputMethodEventComposed(t, e.getCommittedCharacterCount()),
-                t.substring(0, e.getCommittedCharacterCount()),
-                insertionIndex);
+        // TODO: temporarily removed as this is internal API
```

Comment:
Fortunately with the migration to javafx in the maintable-beta branch #3621 this class is no longer needed. So you can leave it like that